NEWS.md

```diff
@@ -80,6 +80,15 @@ example, `">2.5 x ULN"` changed to `">2.5"` for grade 3. (#2534)
 </details>
 
 # admiral 1.1.1
+
+- `check_type = "warning"` default argument added to `derive_param_tte` with an
+  `arg_match` function within the function so the user can use a valid input of 
+  `error, message, warning, or none`. `signal_duplicate_records()` has also been
+  added to the function on lines 394 and 411 to check for uniqueness of records. (#2481)
+  
+- `order()` function has been added to `event_source()` and `censor_source()` and
+  defaulted to `NULL` to allow sorting of input data. (#2481)
```

Comment:
```suggestion
- `order` argument has been added to `event_source()` and `censor_source()` and
  defaulted to `NULL` to allow specifying variables in addition to the date variable. 
  This can be used to ensure the uniqueness of the select records if there is more
  than one record per date.  (#2481)
```